.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "ZMK Development",
+  "dockerFile": "Dockerfile",
```

Comment:
It isn't easily found in the docs, but you can just replace this with:

```suggestion
  "image": "zmkfirmware/zephyr-west-action-arm",
```

And remove the extraneous `Dockerfile`.

The other thing I had to do locally here, to make this work on my Fedora machine w/ SELinux enforcement enabled, was to *also* add a line:

```
"runArgs": ["--security-opt", "label=disable"],
```

Does it work for you locally if you also have that in this file?

Reply:
Yup, fixed in 164cd97

Reply:
Ended up returning to the dockerfile approach so we could use it to install SSH and GPG.